### PR TITLE
Added implementation of feature_permutation

### PR DESCRIPTION
**Pull request by andersbogsnes**
Working on issue #59

### CODE REVIEW

**Comment by thomasfrederikhoeck** [comment]
This implementation can be made to be multi-threaded. I can give it at shot at a later point.

**Comment by thomasfrederikhoeck** [comment]
As this is similar to ELI5's PermutationImportance I suggest the following features that it has:

- Multiple permutation of the same feature to improve estimate of importance.

- Option for Cross-Calidation if model is refitted. This can be used to show how important the features are for generalization.

File: src/ml_tooling/metrics.py [hunk 1]
```diff
@@ -70,6 +71,44 @@ def confusion_matrix(y_true: np.ndarray, y_pred: np.ndarray, normalized=True) ->
...
 
 
+def permuted_feature_importance(model,
+                                x,
+                                y,
+                                refit=True,
+                                metric=None):
+    """
+    Calculates feature importance by randomly permuting features and comparing result to baseline
+    :param y:
```
Comment (thomasfrederikhoeck) [inline src/ml_tooling/metrics.py]: Isn't y training/test target?

**Comment by andersbogsnes** [comment]
I'll have a look at their implementation - see if there's some cleverness we can use

File: src/ml_tooling/metrics.py (hunk 1 above)
Comment (andersbogsnes) [reply src/ml_tooling/metrics.py]: Yeah, changed the implementation without updating docs - still have some work to do on this feature 😄
